docs/ref/contrib/gis/db-api.txt

```diff
@@ -374,38 +374,38 @@ Database functions
 The following table provides a summary of what geography-specific database
 functions are available on each spatial backend.
 
-====================================  =======  ======  ===========  ==========
-Function                              PostGIS  Oracle  MySQL        SpatiaLite
-====================================  =======  ======  ===========  ==========
-:class:`Area`                         X        X       X            X
-:class:`AsGeoJSON`                    X                             X
-:class:`AsGML`                        X        X                    X
-:class:`AsKML`                        X                             X
-:class:`AsSVG`                        X                             X
-:class:`BoundingCircle`               X
-:class:`Centroid`                     X        X       X            X
-:class:`Difference`                   X        X       X (≥ 5.6.1)  X
-:class:`Distance`                     X        X       X (≥ 5.6.1)  X
-:class:`Envelope`                     X                X            X
+====================================  =======  ==============  ===========  ==========
+Function                              PostGIS  Oracle          MySQL        SpatiaLite
+====================================  =======  ==============  ===========  ==========
+:class:`Area`                         X                        X            X
+:class:`AsGeoJSON`                    X                                     X
+:class:`AsGML`                        X                                     X
```

Comment:
Here: https://github.com/django/django/pull/7697 (for master and 1.10). I'll "revert" it when editing and merging https://github.com/django/django/pull/7695.